supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -465,6 +465,15 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*: xref:business-rule[business rule], xref:business-process[business process], xref:decision-table[decision table], xref:data-model[data model], xref:dsl[DSL]
 
+[discrete]
+[[assisted-installer]]
+==== image:images/yes.png[yes] Assisted Installer (noun)
+*Description*: The Assisted Installer provides software-as-a-service functionality for user-friendly cluster installations by using the Red Hat Hybrid Cloud Console. 
```

Comment:
```suggestion
*Description*: In Red Hat OpenShift, the Assisted Installer provides Software-as-a-Service functionality for user-friendly cluster installations by using the Red Hat Hybrid Cloud Console. 
```

Reply:
Two minor tweaks for consistency with other entries:

- Prefix with "In Red Hat OpenShift," for consistency with the **installer-provisioned infrastructure** and **user-provisioned architecture** entries.
- Use initcap for _Software_ and _Service_ for consistency with the **SaaS** entry.

Reply:
Implemented these fixes, thanks again!